src/setup.js

```diff
@@ -32,6 +32,14 @@ module.exports = function(ctx) {
 
       ctx.ui.addButtons();
 
+      if (ctx.options.boxSelect) {
+        map.boxZoom.disable();
+        // Need to toggle dragPan on and off or else first
+        // dragPan disable attempt in simple_select doesn't work
+        map.dragPan.disable();
+        map.dragPan.enable();
+      }
```

Comment:
This ☝️ is weird. If I don't include the dragPan disable/enable toggle, map.dragPan.disable() will not work the first time it's called in simple_select. It will work fine all subsequent times.